sources/packages/api/src/services/education-program-offering/education-program-offering.service.ts

```diff
@@ -25,7 +26,9 @@ import {
   PaginationOptions,
   PaginatedResults,
   getISODateOnlyString,
+  CustomNamedError,
 } from "../../utilities";
+export const OFFERING_NOT_VALID = "OFFERING_NOT_FOUND";
```

Comment:
Please move this to the `error-code.constants.ts`. Both API and Web projects have the dedicated file.